src/internal/pfsdb/common.go

```diff
@@ -20,6 +22,24 @@ type (
 	ModelType interface{ Repo | Commit | Branch }
 )
 
+type OrderByColumn struct {
```

Comment:
This was a great idea although I don't think we have a use case for it yet.

Reply:
What do you mean?